jablib/src/main/java/org/jabref/logic/cleanup/CleanupWorker.java

```diff
@@ -44,8 +51,9 @@ public List<FieldChange> cleanup(CleanupPreferences preset, BibEntry entry) {
     private List<CleanupJob> determineCleanupActions(CleanupPreferences preset) {
         List<CleanupJob> jobs = new ArrayList<>();
 
+        // Add active jobs from preset panel
         for (CleanupPreferences.CleanupStep action : preset.getActiveJobs()) {
-            jobs.add(toJob(action));
+                jobs.add(toJob(action));
         }
```

Comment:
Reads like AI-modified content.

--> indent is wrong and comment could be removed

Could also be done in a follow-up PR